src/js/video.js

```diff
@@ -99,6 +99,11 @@ function videojs(id, options, ready) {
     throw new TypeError('The element or ID supplied is not valid. (videojs)');
   }
 
+  // Check if element is included in the DOM
+  if (Dom.isEl(tag) && !document.body.contains(tag)) {
+    log.warn('The element supplied is\'t included in the DOM');
```

Comment:
This works, though, we tend to switch to double quotes (`"`) if we're including single quotes (`'`) in the text, that way we don't need to escape it :)

Reply:
@gkatsev Thank you. Of course it would look better.
Now I also can see misprint in "isN't".
What should I do with that? Make another commit?

Reply:
If you push another commit to your branch the PR will update.